spec/requests/admin/organizations_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Admin::OrganizationsController, type: [:request, :admin] do
```

Comment:
RSpec/FilePath: Spec path should end with `admin/organizations_controller*_spec.rb`. (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FilePath)